### PR TITLE
Refacto check of model class by creating a controller trait

### PR DESCRIPTION
**Pull request by mcsky**
This check is done in all Crud Controllers. Let's create a trait to avoid duplicate code :)

```php
empty($modelClass) || !class_exists($modelClass) || !is_subclass_of($modelClass, CrudableModelInterface::class);
```

### CODE REVIEW

File: src/Crud/Controller/CrudControllerTrait.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?php
+declare(strict_types=1);
+
+namespace Biig\Melodiia\Crud\Controller;
+
+use Biig\Melodiia\Crud\CrudableModelInterface;
+
+trait ControllerTrait
```
Comment (Nek-) [inline src/Crud/Controller/CrudControllerTrait.php]: Naming it CrudControllerTrait would be better. Or ApiControllerTrait

File: src/Crud/Controller/CrudControllerTrait.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+{
+    /**
+     * @param string $modelClass
+     * @return bool
+     */
+    public function isModelClassInvalid(string $modelClass): bool
```
Comment (Nek-) [inline src/Crud/Controller/CrudControllerTrait.php]: Naming it `assertModelClassInvalid ` and throw the exception inside would be great isn't it?

**Review by Nek- — CHANGES_REQUESTED** [review]
You need to rebase your work, it was based on a previous merged branch.

File: src/Crud/Controller/CrudControllerTrait.php (hunk 1 above)
Comment (mcsky) [reply src/Crud/Controller/CrudControllerTrait.php]: I had it in local actually 😂 It's updated

**Comment by mcsky** [comment]
@Nek- rebase done

**Pull request merged**